content/analytics-and-automation/inclass-activity/_index.md

```diff
@@ -3,6 +3,17 @@ title: "Exercise Instructions"
 weight: 112
 ---
 
-## In Class Activity : Dynatrace UI Tour 
-In this activity, you will learn how to navigate the Dynatrace UI with the provided Dynatrace tutorials. 
-Learners should also take time to familiarize themselves with the tools capabilities to ensure they are able to navigate the documentation for questions and answers. Simaliar to the previous activity on Day 11,  activity is designed for awareness to the Dynatrace interface. The modules in this training will become outdated but Dynatrace will maintain the latest documents, updates and software upgrades and learners are encouraged to review the documentation, communities, user groups outlined. 
\ No newline at end of file
+In this activity, you will begin to familiarize yourself with the vast Dynatrace platform by finding the answers to certain questions within the documentation.
+
+## Scavenger Hunt
+
+Dynatrace has lots of different services and possible configurations to meet a company's needs. Time to test your research skills to find the answers to how the platform can best serve you in a customer service context. Make note of the following answers so you are ready to discuss them as a class!
+
+1. Davis is the name of the AI used to analyze a corporation's enterprise system. It can evaluate commonly occurring issues. In the diagram illustrating green, red, and yellow issues, what does yellow mean?
+1. OneAgent and ActiveGate are in the same section of the documentation. What is the difference?
```

Comment:
Do we need to distinguish between OneAgent and Dynatrace OneAgent?

Dynatrace OneAgent is located within the "Deploy" section along with ActiveGate but there is also a "OneAgent" located under the "Platform" section.

Reply:
Done! Thanks!